components/elements/cards/Card/Card.vue

```diff
@@ -1,3 +1,7 @@
+<script setup lang="ts">
+// blargh
+</script>
```

Comment:
Do we need these stubs in order for things to build/work?

Reply:
Ya, if we don't then it will have a bit of trouble at Test time. These of course because nothing is in them do not effect performance for an end user